front/pages/edit.vue

```diff
@@ -783,7 +804,7 @@ export default {
         },
       });
       const body = await res.json();
-      return body.cooldown;
+      return body.cooldown * 60 * 1000;
```

Comment:
what is the purpose of this change ?

Reply:
It's to convert the returned time from minutes to milliseconds.
Would it be better to have it done backend-side?